### PR TITLE
MT bounds

### PR DESCRIPTION
**Pull request by charleskawczynski**
**This is not a real PR.**

Adds tests for MoistThermodynamics bounds

### CODE REVIEW

**Comment by tapios** [comment]
Maybe a better way to summarize error statistics is to compute the temperature error `\Delta T` after saturation adjustment and summarize `rms(\Delta T)`, `max(|\Delta T|)` etc., and plots `\Delta T` vs `T`. If the errors are near freezing, the temperature errors may be small, just not quite small enough to satisfy the convergence criteria.

**Comment by charleskawczynski** [comment]
> Maybe a better way to summarize error statistics is to compute the temperature error `\Delta T` after saturation adjustment and summarize `rms(\Delta T)`, `max(|\Delta T|)` etc., and plots `\Delta T` vs `T`. If the errors are near freezing, the temperature errors may be small, just not quite small enough to satisfy the convergence criteria.

I like this idea, but how can we compute the error if we can't converge to the correct root (`T_correct` in `\Delta T = T_sat_adjust - T_correct`)?

**Comment by tapios** [comment]
> > Maybe a better way to summarize error statistics is to compute the temperature error `\Delta T` after saturation adjustment and summarize `rms(\Delta T)`, `max(|\Delta T|)` etc., and plots `\Delta T` vs `T`. If the errors are near freezing, the temperature errors may be small, just not quite small enough to satisfy the convergence criteria.
> 
> I like this idea, but how can we compute the error if we can't converge to the correct root (`T_correct` in `\Delta T = T_sat_adjust - T_correct`)?

I had in min just using the value `T_sat_adjust` that you get from saturation adjustment after `maxiter` has been reached (even if convergence criteria are not yet satisfied).

File: test/Common/MoistThermodynamics/thermo_state_iter_sensitivity.jl [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  dir = joinpath("output","MoistThermoAnalysis")
+
+  for method in (PhaseEquil_NewtonMethod, PhaseEquil)
+    for (p, maxiter) in enumerate(maxiter_range)
+    for (i,RH) in enumerate(RH_range)
+    for (j,ρ) in enumerate(ρ_range)
+    for (k,T) in enumerate(T_range)
+      # q_pt = PhasePartition(q_tot,FT(0),q_tot)s
+      q_sat = q_vap_saturation(T, ρ)
+      q_tot = RH*q_sat
```
Comment (tapios) [inline test/Common/MoistThermodynamics/thermo_state_iter_sensitivity.jl]:
```suggestion
      q_tot = min(RH*q_sat, 1)
```

**Comment by charleskawczynski** [comment]
This will be moved to [MoistThermodynamics.jl](https://github.com/climate-machine/MoistThermodynamics.jl) once PlanetParameters are replaced with CLIMAParameters.

**Pull request closed without merging**